1-prep/1-compiling.md

```diff
@@ -47,8 +47,13 @@ cd polkadot
 # Checkout the proper commit
 git checkout 93f0029
 
+# Setup proper Rust version for compiling this workshop
+rustup uninstall nightly
```

Comment:
```suggestion

```
You don't need to uninstall nightly if you specify the correct version in the `cargo +nightly-2020-10-06 build` command